### PR TITLE
Yotam/add db

### PR DESCRIPTION
**Pull request by yotamc4**
added entities,
created db with tables,
added implementing classes for managers

current data model:
image

### CODE REVIEW

File: ServerSide/yoty/Migrations/YotyContextModelSnapshot.cs [hunk 1]
```diff
@@ -0,0 +1,268 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Yoty.Data;
+
+namespace yoty.Migrations
```
Comment (tzachihakmon) [inline ServerSide/yoty/Migrations/YotyContextModelSnapshot.cs]: should be YOTY.Service.Data.Migration

File: ServerSide/yoty/Migrations/20201119142149_init.cs [hunk 2]
```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace yoty.Migrations
```
Comment (tzachihakmon) [inline ServerSide/yoty/Migrations/20201119142149_init.cs]: YOTY.Service.Data.Migration

File: ServerSide/yoty/Managers/SellerManager.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using YOTY.Service.Managers;
+
+namespace yoty.Managers
```
Comment (tzachihakmon) [inline ServerSide/yoty/Managers/SellerManager.cs]: YOTY.Service.Managers

File: ServerSide/yoty/Managers/ProductsBidsManager.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using YOTY.Service.Managers;
+
+namespace yoty.Managers
```
Comment (tzachihakmon) [inline ServerSide/yoty/Managers/ProductsBidsManager.cs]: YOTY.Service.Managers

File: ServerSide/yoty/Migrations/YotyContextModelSnapshot.cs (hunk 1 above)
Comment (yotamc4) [reply ServerSide/yoty/Migrations/YotyContextModelSnapshot.cs]: @tzachihakmon right, I’ll fix all namespaces

File: ServerSide/yoty/Managers/SellerManager.cs (hunk 3 above)
Comment (yotamc4) [reply ServerSide/yoty/Managers/SellerManager.cs]: Done

File: ServerSide/yoty/Migrations/20201119142149_init.cs (hunk 2 above)
Comment (yotamc4) [reply ServerSide/yoty/Migrations/20201119142149_init.cs]: Done

File: ServerSide/yoty/Managers/ProductsBidsManager.cs (hunk 4 above)
Comment (yotamc4) [reply ServerSide/yoty/Managers/ProductsBidsManager.cs]: Done

File: .vs/ProjectSettings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (tzachihakmon) [inline .vs/ProjectSettings.json]: delete

File: .vs/VSWorkspaceState.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (tzachihakmon) [inline .vs/VSWorkspaceState.json]: delete

File: ServerSide/yoty/YOTY.Service.csproj [hunk 7]
```diff
@@ -1,8 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (tzachihakmon) [inline ServerSide/yoty/YOTY.Service.csproj:4]: delete

**Pull request merged**